examples/event-counters/Examples.EventCounters/Examples.EventCounters.csproj

```diff
@@ -0,0 +1,19 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net6.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="OpenTelemetry" Version="1.3.1" />
+    <PackageReference Include="OpenTelemetry.Exporter.Console" Version="1.3.1" />
```

Comment:
In #857 
OpenTelemetryCoreLatestVersion was introduced for examples.
It will be good to follow this pattern